src/ontology/cl-edit.owl

```diff
@@ -31920,6 +31921,17 @@ AnnotationAssertion(rdfs:label obo:CL_4040005 "mesenchymal stem cell of apical p
 SubClassOf(obo:CL_4040005 obo:CL_0000134)
 SubClassOf(obo:CL_4040005 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_0003677))
 
+# Class: obo:CL_4042001 (TAC3-positive interneuron)
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:34727523") obo:IAO_0000115 obo:CL_4042001 "A GABAergic interneuron that has its soma in the striatum and that has an enriched expression of the gene TAC3.")
+AnnotationAssertion(<http://purl.org/dc/terms/contributor> obo:CL_4042001 <https://orcid.org/0000-0002-0098-8958>)
+AnnotationAssertion(<http://purl.org/dc/terms/date> obo:CL_4042001 "2024-03-05T14:29:32Z"^^xsd:dateTime)
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "  PMID:34727523") oboInOwl:hasExactSynonym obo:CL_4042001 "TAC3 interneuron ")
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:32999462") Annotation(oboInOwl:hasDbXref "PMID:34727523") rdfs:comment obo:CL_4042001 "In Human, Rhesus Macaque, and Marmoset, TAC3-positive interneuron has its soma in the striatum.")
```

Comment:
Does it mean that in other species the soma is not in the striatum? If that is the case, it should be removed from the textual definition and the axioms.

Reply:
I added "in Human, Rhesus Macaque, and Marmoset" specifically because the TAC3-positive interneurons have been observed only in those species. In [PMID:32999462](https://doi.org/10.1038/s41586-020-2781-z) they detect TAC3-positive interneurons in human and marmoset but not in rodents. It seems that in other species this cell type has not been detected (so far).

specifically these interneurons are described in Rhesus Macaque in [PMID:34727523](https://doi.org/10.1016/j.cub.2021.10.015), and in Marmoset and Human in [PMID:32999462](https://doi.org/10.1038/s41586-020-2781-z). The latter specify the absence of TAC3-positive interneurons in other species: 

>The abundance of the previously undescribed TAC3+ interneuron population raised the question of whether it had replaced—or supplemented—conserved interneuron populations. The primates had an expanded representation of interneurons in the striatum overall: whereas interneurons consisted of 5.1% of all striatal neurons in the ferret and 4.1% of all striatal neurons in mice, they consisted of 13.1% of all striatal neurons in marmosets and 10.8% in humans, which is consistent with the stereological estimates of higher interneuron proportions in the primate striatum[43](https://www.nature.com/articles/s41586-020-2781-z#ref-CR43). Thus, compared to ferrets and mice, the primate striatum has more than doubled the proportion of interneurons, in substantial part by introducing a previously undescribed interneuron type that has no clear counterpart in the mouse or ferret.

Provided the references, let me know whether it is best to exclude the rdfs:comment